examples/server/assets/js/socket.js

```diff
@@ -1,11 +1,14 @@
 // NOTE: The contents of this file will only be executed if
 // you uncomment its entry in "assets/js/app.js".
 
-// To use Phoenix channels, the first step is to import Socket
-// and connect at the socket path in "lib/web/endpoint.ex":
+// To use Phoenix channels, the first step is to import Socket,
+// and connect at the socket path in "lib/web/endpoint.ex".
+//
+// Pass the token on params as below. Or remove it
+// from the params if you are not using authentication.
 import {Socket} from "phoenix"
 
-let socket = new Socket("/socket", {})
+let socket = new Socket("/socket", {params: {token: window.userToken}})
```

Comment:
It feels a bit strange to have added the auth process here.  For an example application I'd suggest that it be left as default.

Reply:
ok!